[2022-08-04 11:25:57,428] {processor.py:153} INFO - Started process (PID=54064) to work on /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:25:57,428] {processor.py:641} INFO - Processing file /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-08-04 11:25:57,428] {logging_mixin.py:115} INFO - [2022-08-04 11:25:57,428] {dagbag.py:508} INFO - Filling up the DagBag from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:25:57,434] {logging_mixin.py:115} INFO - [2022-08-04 11:25:57,434] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-08-04 11:25:57,434] {logging_mixin.py:115} INFO - [2022-08-04 11:25:57,434] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-08-04 11:25:57,434] {processor.py:653} WARNING - No viable dags retrieved from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:25:57,438] {processor.py:161} INFO - Processing /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.011 seconds
[2022-08-04 11:26:29,380] {processor.py:153} INFO - Started process (PID=54253) to work on /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:26:29,386] {processor.py:641} INFO - Processing file /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-08-04 11:26:29,386] {logging_mixin.py:115} INFO - [2022-08-04 11:26:29,386] {dagbag.py:508} INFO - Filling up the DagBag from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:26:29,394] {logging_mixin.py:115} INFO - [2022-08-04 11:26:29,394] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-08-04 11:26:29,394] {logging_mixin.py:115} INFO - [2022-08-04 11:26:29,394] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-08-04 11:26:29,394] {processor.py:653} WARNING - No viable dags retrieved from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:26:29,400] {processor.py:161} INFO - Processing /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2022-08-04 11:27:04,096] {processor.py:153} INFO - Started process (PID=54334) to work on /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:27:04,102] {processor.py:641} INFO - Processing file /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-08-04 11:27:04,102] {logging_mixin.py:115} INFO - [2022-08-04 11:27:04,102] {dagbag.py:508} INFO - Filling up the DagBag from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:27:04,111] {logging_mixin.py:115} INFO - [2022-08-04 11:27:04,111] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-08-04 11:27:04,111] {logging_mixin.py:115} INFO - [2022-08-04 11:27:04,111] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-08-04 11:27:04,111] {processor.py:653} WARNING - No viable dags retrieved from /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-08-04 11:27:04,117] {processor.py:161} INFO - Processing /home/ndx/Innowise tasks/Innowise_task_5/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
